packages/components/src/components/button/button.css

```diff
@@ -126,8 +126,10 @@
 }
 
 .button.button--icon-before:not(.button--icon-only) ::slotted(*) {
-  margin-right: var(--spacing-icon-x);
-  margin-left: calc(var(--spacing-icon-x) * -1);
+  margin-right: 8px;
```

Comment:
i think it would be better here to update the value of --spacing-icon-x instead of hard coding `8px`, unless there is some reason that I am overlooking...